cmd/repl/repl.go

```diff
@@ -240,6 +241,7 @@ func getDataSource(dataSource, db string) string {
 var currentDB string
 
 func main() {
+	step.InitEnvFromFile(filepath.Join(os.Getenv("HOME"), step.DotEnvFilename))
```

Comment:
Seems this file needs to be created in every step container but it seems Couler can not do that now.